app/src/main/res/layout/fragment_monster_list.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.recyclerview.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/monster_list_recyclerview"
+    android:name="com.theuhooi.uhooipicbook.monsterlist.MonsterListFragment"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_marginStart="@dimen/left_margin"
+    android:layout_marginEnd="@dimen/right_margin"
+    app:layoutManager="LinearLayoutManager"
```

Comment:
Useless `layoutManager` definition because you have set `GridLayoutManager` programatically.

Reply:
Oh😅
Changed to set GridLayoutManager to XML.